code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -218,8 +218,8 @@
 	new /obj/item/weapon/reagent_containers/glass/bottle/polonium(src)
 	new /obj/item/weapon/reagent_containers/glass/bottle/venom(src)
 	new /obj/item/weapon/reagent_containers/glass/bottle/neurotoxin2(src)
-	new /obj/item/weapon/reagent_containers/glass/bottle/formaldehyde(src)
-	new /obj/item/weapon/reagent_containers/glass/bottle/spewium(src)
+	new /obj/item/weapon/reagent_containers/glass/bottle/formaldehyde(src)
+	new /obj/item/weapon/reagent_containers/glass/bottle/spewium(src)
```

Comment:
I don't know why it says these are changed, either. I definitely didn't touch them